xbmc/utils/ActorProtocol.h

```diff
@@ -51,7 +51,9 @@ class Message
   bool Reply(int sig, void *data = NULL, int size = 0);
 
 private:
-  Message() {isSync = false; data = NULL; event = NULL; replyMessage = NULL;};
+  Message()
+  : signal(0), isSync(false), isSyncFini(false), isOut(false), isSyncTimeout(false),
```

Comment:
My intention was ofc not to initialize instances with random values. ;-) For sure the right initial values should be chosen. Well change isOut imo value. Thanks for bringing this up.